source/classes.tex

```diff
@@ -1387,12 +1387,11 @@
 
 \rSec2[class.nest]{Nested class declarations}%
 \indextext{definition!nested class}%
-\indextext{class local|see{local class}}%
-\indextext{class nested|see{nested class}}
+\indextext{class!nested|see{nested class}}
 
 \pnum
 A class can be declared within another class. A class declared within
-another is called a \grammarterm{nested} class. The name of a nested class
+another is called a \defnx{nested}{nested class} class. The name of a nested class
 is local to its enclosing class.
```

Comment:
The defined phrase is actually "nested class", so I'd like to see both words in italics. (Same for "local class".)